### ISSUE TITLE
Upgrade doc for 1.0 contains broken link

### ISSUE THREAD

**Issue by msonnleitner** [crossplane/crossplane#2043]
### What happened?
The upgrade guide on
https://github.com/crossplane/crossplane/blob/release-1.0/docs/guides/upgrading-to-v1.0.md
references an url which returns 404:

https://raw.githubusercontent.com/crossplane/crossplane/release-1.0/cluster/charts/crossplane/crds

**Comment by hasheddan** [comment]
Thanks @msonnleitner! Fix is open in #2044

**Comment by msonnleitner** [comment]
@hasheddan it is maybe a minor inconvenience, but when upgrading to the 1.0.0 version, the 
helm repo has also to be added (which is mentioned in v1 installation guide) 

```
helm repo add crossplane-stable https://charts.crossplane.io/stable
helm repo update
```
